### ISSUE TITLE
Error building in Raspberry Pi

### ISSUE THREAD

**Issue by sharad461** [google/sentencepiece#449]
![Untitled](https://user-images.githubusercontent.com/24848946/73626595-926bab80-4671-11ea-84af-934046e9dcdb.png)

While making sentencepiece C++ on a Raspberry Pi 4 with 4 GB RAM, this is the error I get. I'm using the instructions in the readme file to build.

``` uname -a ```:
Linux raspberrypi 4.19.93-v7l+ #1290 SMP Fri Jan 10 16:45:11 GMT 2020 armv7l GNU/Linux

What am I doing wrong? 
Can provide more details if necessary.

**Comment by sharad461** [comment]
Not very experienced with making, so instead I developed a workaround, and am mentioning it here for people who find themselves in a similar situation.

I made my computer (that easily installs sentencepiece) an SSH server, wrote a script in it that does the encodings and decodings, and used `fabric` package to run these scripts remotely from the Raspberry Pi. I made the Raspberry Pi an access point and connected the computer to it. 

Closing.
